### PR TITLE
Fix/MS-844/highlighter in fire fox

### PR DESCRIPTION
**Pull request by dresha-dev**
**Related to task:**  https://oat-sa.atlassian.net/browse/MS-844
**Description:** Fix scoring highlighter in Firefox
**How to test:** 
- Login as scorer in FF browser
- Try to use highlighter

*Current behavior:* The feature is not working in FF

### CODE REVIEW

File: views/js/previewer/plugins/content/scoringHighlighter.js [hunk 1]
```diff
@@ -108,9 +107,9 @@ define([
             function getAllRanges(selection) {
                 const allRanges = [];
 
-                for (let i = 0; i < selection.rangeCount; i++) {
-                    allRanges.push(selection.getRangeAt(i));
-                }
+                    for (let i = 0; i < selection.rangeCount; i++) {
```
Comment (ampaveliev) [inline views/js/previewer/plugins/content/scoringHighlighter.js:110]: extra indent :thinking:

**Review by ampaveliev — APPROVED** [review]
Looks fine. I can't test it because local MS env doesn't work in the FF

**Review by anshul119 — APPROVED** [review]
works well on local!

File: views/js/previewer/plugins/content/scoringHighlighter.js (hunk 1 above)
Comment (dresha-dev) [reply views/js/previewer/plugins/content/scoringHighlighter.js:110]: oh, I forgot to run prettier for that project 😔

**Pull request merged**